[VM] Update scale set tutorial for scalesetid from PR 6627

iainfoulds opened the pull request.

azurecla commented:
Hi __@v-cmans__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-cmans). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

pranoybej committed:
Updated the "Configure autoscale rules" PowerShell script for Simplicity

pranoybej committed:
Update tutorial-create-vmss.md

I have removed the unnecessary content from line no 225-272 as per your suggestion.
Thanks for pointing out.

pranoybej committed:
Update tutorial-create-vmss.md

pranoybej committed:
Update tutorial-create-vmss.md

iainfoulds committed:
[VM] Adding div back into 'Next steps' section
